.github/workflows/smoke-test.yml

```diff
@@ -48,25 +48,29 @@ jobs:
         # fix formatting with: cargo fmt --manifest-path=rust/Cargo.toml --all
         run: cargo fmt --manifest-path=Cargo.toml --all --check
 
-      - name: Build
-        run: cargo build --verbose --features untrusted,bindings,polars
-      
       - name: Check --no-default-features
-        run: cargo check --verbose --no-default-features --features untrusted,ffi,polars
-
-      - name: Test
-        run: cargo test --verbose --features untrusted,ffi,polars
+        run: cargo check --verbose --no-default-features --features untrusted,bindings
```

Comment:
This isn't critical to the fix, right? We'd still expect any combination of features to build, but as a quick check, we're dropping ffi to get some failures faster?

Reply:
I reordered the arguments so that the builds would mirror each other. Pulled some checks to the top.